packaging/msi/verify_msi.cmd

```diff
@@ -0,0 +1,38 @@
+:: Copyright 2021 Tetrate
+::
+:: Licensed under the Apache License, Version 2.0 (the "License");
+:: you may not use this file except in compliance with the License.
+:: You may obtain a copy of the License at
+::
+::     http://www.apache.org/licenses/LICENSE-2.0
+::
+:: Unless required by applicable law or agreed to in writing, software
+:: distributed under the License is distributed on an "AS IS" BASIS,
+:: WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+:: See the License for the specific language governing permissions and
+:: limitations under the License.
+
+:: verify_msi is written in cmd because msiexec doesn't agree with git-bash
+:: See https://github.com/git-for-windows/git/issues/2526
+@echo off
+if not defined MSI_FILE set MSI_FILE=dist\func-e_windows_amd64\func-e.msi
+:: this is how you do MSI_PRODUCT_CODE=$(command) in cmd:
```

Comment:
I put this in originally to verify it works. I'll find another place to do verification which isn't so crufty. probably set GH env in msi.yaml (using powershell). After that, this cmd file will be easier on the eyes